SwiftCompilerSources/Sources/SIL/Instruction.swift

```diff
@@ -1000,8 +1053,12 @@ final public class DestructureTupleInst : MultipleValueInstruction, UnaryInstruc
 
 final public class BeginApplyInst : MultipleValueInstruction, FullApplySite {
   public var numArguments: Int { bridged.BeginApplyInst_numArguments() }
-
+  
```

Comment:
added space

Reply:
ok I got this one. But I generally don't take time to edit auto-inserted spaces. When we have a precheckin swift-format script that works, I'll use that.